.github/workflows/test_docker_compose_acceptance.yml

```diff
@@ -0,0 +1,138 @@
+# Also note that this name *MUST* match the filename because GHA
+# only provides the workflow name (https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables)
+# and GH APIs only support querying by workflow *FILENAME* (https://developer.github.com/v3/actions/workflows/#get-a-workflow)
+name: docker-compose-acceptance
+on:
+  schedule:
+    - cron: '* * * * *'
```

Comment:
we should start much less aggressive than once per minute -- this will chew through all of our parallel workers very very quickly

let's start hourly or half-hourly and ramp it up from there

Reply:
changed to every 30 min